cmd/clusterctl/examples/vsphere/generate-yaml.sh

```diff
@@ -52,6 +57,7 @@ while test $# -gt 0; do
             echo "options:"
             echo "-h, --help                show brief help"
             echo "-f, --force-overwrite     if file to be generated already exists, force script to overwrite it"
+            echo "--kubecon					Use the containers from the kubecon demo"
```

Comment:
cosmetic: misaligned with the other help and the other help text starts with lower-case.

Reply:
I'll fix that in a future PR